package.json

```diff
@@ -68,7 +67,8 @@
     "stream-browserify": "^1.0.0",
     "ttag": "^1.7.24",
     "unleash-proxy-client": "^1.11.0",
-    "url": "^0.11.0"
+    "url": "^0.11.0",
+    "vision-camera-code-scanner": "^0.2.0"
```

Comment:
From what I could find, we could either:
- [Implement the code reader ourselves](https://www.dynamsoft.com/codepool/react-native-qr-code-scanner-vision-camera.html)
- Use the `vision-camera-code-scanner` (solution proposed by this PR)
- Use the [expo camera](https://docs.expo.dev/versions/latest/sdk/camera/), which requires installing the `expo` modules

I would be interested in using the expo modules, since they seem to be the most recommended way to develop on React Native nowadays, but this would probably be the subject of a whole other investigation / project.

So it looks like using this code scanner is the best approach for now - and we could always fork it / copy its contents to our repo in the future if needed, since it has a license that allows it.

Reply:
I think that if we can ensure its working on the latest RN version we can use it without concerns and later reassess this decision.